tests/src/Traits/RequestModelTestTrait.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\Tests\CdtClient\Traits;
+
+use OpenEuropa\CdtClient\Model\Request\Callback;
+use OpenEuropa\CdtClient\Model\Request\CallbackCollection;
+use OpenEuropa\CdtClient\Model\Request\File;
+use OpenEuropa\CdtClient\Model\Request\ReferenceFile;
+use OpenEuropa\CdtClient\Model\Request\ReferenceFileCollection;
+use OpenEuropa\CdtClient\Model\Request\ReferenceUrl;
+use OpenEuropa\CdtClient\Model\Request\ReferenceUrlCollection;
+use OpenEuropa\CdtClient\Model\Request\SourceDocument;
+use OpenEuropa\CdtClient\Model\Request\SourceDocumentCollection;
+use OpenEuropa\CdtClient\Model\Request\Translation;
+use OpenEuropa\CdtClient\Model\Request\TranslationJob;
+use OpenEuropa\CdtClient\Model\Request\TranslationJobCollection;
+
+/**
+ * Trait RequestModelTestTrait
+ *
+ * Provides helper methods for testing classes that utilize the request data models.
+ */
+trait RequestModelTestTrait
```

Comment:
This trait was added to make it easier to generate request data. By default, it returns proper values that pass the validation.